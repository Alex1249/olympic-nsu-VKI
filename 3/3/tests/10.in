15 6
0.123456
