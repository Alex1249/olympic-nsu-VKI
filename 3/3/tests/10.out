0.1cb9e6
